features/comments/commenting.feature

```diff
@@ -27,6 +27,34 @@ Feature: Commenting
     And I should see "Hello from Comment"
     And I should see a comment by "admin@example.com"
 
+  Scenario: Create an anonymous comment without authentication
+    Given a show configuration of:
+      """
+        # without :required options for rails 5
+        if Rails::VERSION::MAJOR >= 5
+          # assume Rails.application.config.active_record.belongs_to_required_by_default = true
+          ActiveAdmin::Comment.belongs_to_required_by_default = true
+          ActiveAdmin::Comment.belongs_to :author, polymorphic: true
+        else # use required: true for rails 4
+          ActiveAdmin::Comment.belongs_to :author, polymorphic: true, required: true
+        end
+
+        # diable authentication
```

Comment:
Small typo, s/diable/disable

Reply:
you got sharp eyes, bro. I suggest updating these in future updates.